src/sasData.js

```diff
@@ -161,8 +161,8 @@ SasData.prototype.addTable = function(table, macroName, specs) {
         if(val === true || val === false) {
           throw new h54sError('typeError', 'Boolean value in one of the values (columns) is not allowed');
         }
-        if(type === 'string' && val.indexOf('\n') !== -1) {
-          throw new h54sError('typeError', 'New line character is not supported');
+        if(type === 'string' && val.indexOf('\r\n') !== -1) {
```

Comment:
Let's change this so that \r\n is replaced with a \n, that way all newlines are supported